src/nomad_simulations/schema_packages/properties/integrals.py

```diff
@@ -0,0 +1,150 @@
+from typing import TYPE_CHECKING
+
+import numpy as np
+from nomad.metainfo import MEnum, Quantity
+
+if TYPE_CHECKING:
+    from nomad.datamodel.datamodel import EntryArchive
+    from nomad.metainfo import Context, Section
+    from structlog.stdlib import BoundLogger
+
+from nomad_simulations.schema_packages.physical_property import PhysicalProperty
+
+
+class OneElectronIntegral(PhysicalProperty):
```

Comment:
Given that you are following a popular standard (which I applaud) could we maybe integrate both further via our `HDF5Reference`? This way, the user provides a TREXIO-compliant HDF5 file (or we generate it from its `.txt`-form), and directly connect it to our schema.

@ladinesa Any further suggestions / considerations here?

Reply:
TREXIO is a schema specification that comes in HDF5 and text format.
There may of course also be the possibility of reading the necessary data directly from the parsed output.